db/python/layers/web.py

```diff
@@ -372,13 +387,28 @@ async def get_project_summary(
         seen_seq_types = set(cram_number_by_seq_type.keys()).union(
             set(seq_number_by_seq_type.keys())
         )
-        sequence_stats = {}
-        for seq in seen_seq_types:
-            sequence_stats[seq] = {
+        seen_batches = set(
+            [
+                item
+                for s in seen_seq_types
+                for item in seq_number_by_seq_type_and_batch.get(s).keys()
+            ]
+        )
+        sequence_stats: Dict[str, Dict[str, str]] = {}
+        cram_seqr_stats = {}
+
+        for seq in sorted(seen_seq_types):
+            cram_seqr_stats[seq] = {
                 'Sequences': str(seq_number_by_seq_type.get(seq, 0)),
                 'Crams': str(cram_number_by_seq_type.get(seq, 0)),
                 'Seqr': str(seqr_stats_by_seq_type.get(seq, 0)),
             }
+            seq_batch = seq_number_by_seq_type_and_batch.get(seq, 0)
+            sequence_stats[seq] = {}
+            for batch in sorted(
```

Comment:
The sorting doesn't really mean anything here as it's dictionary, if we need sorting we should make it a list.

Reply:
seen_batches and seen_seq_types are sets which are basically lists so sorting should work?

Reply:
The actual sorting makes sense, but you put the sorted result on a dictionary. In the end, I think Python orders the JSON string in the same order as inserted, and Typescript's `Object.keys` probably preserves the "order seen", but this ordering a map, dict, object is undefined.

Reply:
Removed sorting for now, can sort on client later if required